lms/djangoapps/course_home_api/course_metadata/views.py

```diff
@@ -103,8 +105,14 @@ def get(self, request, *args, **kwargs):
             is_course_staff=original_user_is_staff
         )
 
+        # User locale settings
+        user_timezone_locale = user_timezone_locale_prefs(request)
+        user_timezone = user_timezone_locale['user_timezone']
```

Comment:
Since you're already doing this, should we just add in the `user_language` as well? I guess if it's not needed, no reason to, but /shrug